flutter/lib/src/event_processor/screenshot_event_processor.dart

```diff
@@ -56,6 +68,8 @@ class ScreenshotEventProcessor implements EventProcessor {
           rethrow;
         }
       }
+    } else if (shouldDebounce) {
+      return event;
```

Comment:
```suggestion
    } else if (shouldDebounce) {
      _options.logger(
        SentryLevel.debug,
        'Skipping screenshot capture due to debouncing (too many captures within ${_debouncer.waitTimeMs}ms)',
      );
      return event;
```

might make sense to add this